distributed/cli/dask_scheduler.py

```diff
@@ -52,7 +53,9 @@ def main(center, host, port, http_port, bokeh_port, show, _bokeh, bokeh_whitelis
     scheduler = Scheduler(center, ip=ip,
                           services={('http', http_port): HTTPScheduler},
                           loop=loop)
+    remote_client = RemoteClient(ip=ip)
     scheduler.start(port)
+    remote_client.start(port=remote_client_port)
```

Comment:
Another option here is that we could treat `RemoteClient` as a service, similar to how `HTTPScheduler` is handled now.